tests/unit/test_utils.py

```diff
@@ -701,6 +703,30 @@ def test_manylinux_check_glibc_version(self):
                     # Didn't find the warning we were expecting
                     assert False
 
+    def test_glibc_version_string_os_fail(self, monkeypatch):
+
+        if hasattr(os, "confstr"):
```

Comment:
Might not need this, I think the tests below would also work on platforms without `confstr`.